### PR TITLE
📖 Adding missing documentation for Token-Permissions

### PR DESCRIPTION
**Pull request by chrismcgehee**
* **Please check if the PR fulfills these requirements**
- [n/a] Tests for the changes have been added (for bug fixes / features)
- [x] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

* **What is the current behavior?** (You can also link to an open issue here)
The Token-Permissions check documentation does not include documentation on when the score is reduced due to certain write permissions being present on a job.

### CODE REVIEW

**Review by azeemshaikh38 — APPROVED** [review]
Looks good to me. Will also wait for @laurentsimon's approval.

File: docs/checks.md
Comment (laurentsimon) [inline docs/checks.md:584]: do you mind double checking the `actions` permissions (I ought to take a closer look but have not fond the time to do so)? I don't think it allows adding a workflow, for example. I think it allows running an existing workflow, which could by pass the LGTM that is used before running workflows on pull request.

**Review by laurentsimon — APPROVED** [review]

File: docs/checks.md
Comment (justaugustus) [reply docs/checks.md:584]: @chrismcgehee -- Ping on Laurent's q above

Comment (chrismcgehee) [reply docs/checks.md:584]: If https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-actions exhaustively lists what can be done with the `actions:write` permission, then it allows:
* Deleting artifacts from a previous action run
* Approving to run an action that needs approval
* Canceling an action that's currently running
* Deleting all logs from a workflow run
* Re-run a workflow that was previously run

I think approving an action may allow an attacker to steal GitHub secrets. Does that seem like enough to keep it as a high risk permission?

Comment (laurentsimon) [reply docs/checks.md:584]: I'd say we can keep it as high for now, since secrets typically give access to resources like ppg keys, cloud services, etc. If we get complaints of false positives, we could double-check that there is at least one workflow that uses a secret in the repo.
Wdut?

Comment (chrismcgehee) [reply docs/checks.md]: That sounds good. I left the scoring the same but updating the verbiage for the documentation.

**Review by laurentsimon — APPROVED** [review]
Thanks Chris!

**Pull request merged**
